### PR TITLE
Fixed null output schema

### PR DESCRIPTION
**Pull request by psainics**
Don't set output schema to null , if macro is enabled

### CODE REVIEW

File: src/main/java/io/cdap/plugin/snowflake/common/util/SchemaHelper.java [hunk 1]
```diff
@@ -85,6 +81,14 @@ public static Schema getSchema(SnowflakeSourceAccessor snowflakeAccessor, String
     }
   }
 
+  public static Schema getParsedSchema(String schema) {
+    try {
```
Comment (vikasrathee-cs) [inline src/main/java/io/cdap/plugin/snowflake/common/util/SchemaHelper.java:88]: Add null check here as schema can be null, in that case it should return null

Comment (vikasrathee-cs) [reply src/main/java/io/cdap/plugin/snowflake/common/util/SchemaHelper.java:88]: And no need to have schema as input parameter for the method

**Comment by vikasrathee-cs** [comment]
> Don't set output schema to null , if macro is enabled

Add Junits for the same

File: src/main/java/io/cdap/plugin/snowflake/common/util/SchemaHelper.java (hunk 1 above)
Comment (vikasrathee-cs) [reply src/main/java/io/cdap/plugin/snowflake/common/util/SchemaHelper.java:88]: and make the method private

Comment (psainics) [reply src/main/java/io/cdap/plugin/snowflake/common/util/SchemaHelper.java:88]: Updated

**Comment by psainics** [comment]
New Unit Test
- testGetSchemaWhenMacroIsEnabled
- testGetSchemaWhenMacroIsEnabledSchemaIsNull
image

File: src/main/java/io/cdap/plugin/snowflake/common/util/SchemaHelper.java (hunk 1 above)
Comment (psainics) [reply src/main/java/io/cdap/plugin/snowflake/common/util/SchemaHelper.java:88]: > And no need to have schema as input parameter for the method

Didn't understand this request

Comment (vikasrathee-cs) [reply src/main/java/io/cdap/plugin/snowflake/common/util/SchemaHelper.java:88]: > > And no need to have schema as input parameter for the method
> 
> Didn't understand this request

ignore it. I thought it's a class level variable

Comment (vikasrathee-cs) [reply src/main/java/io/cdap/plugin/snowflake/common/util/SchemaHelper.java:88]: squash the commits and raise an external PR

**Pull request merged**